chalice/app.py

```diff
@@ -264,17 +273,40 @@ def __init__(self, body, headers=None, status_code=200):
         if headers is None:
             headers = {}
         self.headers = headers
+        self.case_insensitive_headers = CaseInsensitiveMapping(headers)
         self.status_code = status_code
+        self._is_base64_encoded = False
+
+    def _base64encode(self, data):
+        if not isinstance(data, bytes):
+            raise ValueError('Expected bytes type for body with binary '
+                             'Content-Type. Got %s type body instead.'
+                             % type(data))
+        data = base64.b64encode(data)
+        return data.decode('ascii')
+
+    def encode_as_binary(self):
```

Comment:
Similar to the other comment, not sure if I am missing/making up use cases. But I thought it could be useful in the case where a user wants to return binary data that is received as a base64 string rather than the raw bytes. For example something that is intended to be printed straight to an ascii term. 

Maybe thats a contrived use case?